docker/jupyter-scipy-notebook-ee/test_jupyter_scipy_notebook_ee.ipynb

```diff
@@ -0,0 +1,140 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Summary\n",
+    "\n",
+    "This is a notebook for basic manual testing of the packages installed by the Docker image."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Test ipywidgets\n",
+    "https://github.com/jupyter-widgets/ipywidgets"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import ipywidgets"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ipywidgets.IntSlider()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Test bqplot\n",
+    "\n",
+    "https://github.com/bloomberg/bqplot"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from bqplot import pyplot as plt\n",
+    "import numpy as np\n",
+    "\n",
+    "plt.figure(1, title='Line Chart')\n",
+    "np.random.seed(0)\n",
+    "n = 200\n",
+    "x = np.linspace(0.0, 10.0, n)\n",
+    "y = np.cumsum(np.random.randn(n))\n",
+    "plt.plot(x, y)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Test ipyleaflet\n",
+    "https://github.com/ellisonbg/ipyleaflet"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import ipyleaflet\n",
+    "\n",
+    "ipyleaflet.Map(center=[34.62, -77.34], zoom=10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Test the Earth Engine Python API\n",
+    "\n",
+    "https://github.com/google/earthengine-api"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from IPython.display import Image\n",
+    "import ee\n",
+    "\n",
+    "# Note that you need to authenticate the server before initializing the package\n",
+    "# by running the following command in a terminal: earthengine authenticate\n",
+    "ee.Initialize()\n",
+    "\n",
+    "# Generate a URL that displays a global DEM.\n",
+    "url = ee.Image(\"USGS/SRTMGL1_003\").getThumbUrl({'min':0, 'max':3000})\n",
+    "Image(url=url)"
```

Comment:
Replace by: `Image(url=url, embed=True, format='png')` to make sure preview image is embedded and shown in the repo - useful to preview notebooks directly via github

Reply:
In this case I do not want to save the output in the notebook, because the main purpose of this notebook is to test if you can actually run the notebook and get results.

Reply:
Ah, then LGTM